src/flatland/ordered/set.clj

```diff
@@ -56,7 +56,7 @@
   (toString [this]
     (str "#{" (clojure.string/join " " (map str this)) "}"))
   (hashCode [this]
-    (reduce + (map #(.hashCode ^Object %) (.seq this))))
+    (reduce + (keep #(when % (.hashCode ^Object %)) (.seq this))))
```

Comment:
This doesn't look good to me, since `false` will be treated the same way here as `nil`

Reply:
cc @jafingerhut

Reply:
Good catch.  Highly recomended to add an explicit test case with `false` as an element of a set, to confirm, and then fix the expected issue that such a test case should find.